source/installation-guide/packages-list.rst

```diff
@@ -350,3 +352,63 @@ MSU
+|               |              |     i386     | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386_EL5|.el5.i386.rpm <https://packages.wazuh.com/|CURRENT_MAJOR|/yum5/i386/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386_EL5|.el5.i386.rpm>`__ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386_EL5|.el5.i386.rpm.sha512>`__)         |
++               +--------------+--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |              |              | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86|.x86_64.rpm <|RPM_AGENT_DEBUG_SYMBOLS|-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86|.x86_64.rpm>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_X86|.x86_64.rpm.sha512>`__)                      |
++               +              +    x86_64    +--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |              |              | `wazuh-manager-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_MANAGER_X86|.x86_64.rpm <|RPM_MANAGER_DEBUG_SYMBOLS|-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_MANAGER_X86|.x86_64.rpm>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-manager-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_MANAGER_X86|.x86_64.rpm.sha512>`__)                |
++               +              +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|               |              |     i386     | `wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm <|RPM_AGENT_DEBUG_SYMBOLS|-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm>`_ (`sha512 <|CHECKSUMS_URL||WAZUH_LATEST|/wazuh-agent-debuginfo-|WAZUH_LATEST|-|WAZUH_REVISION_YUM_AGENT_I386|.i386.rpm.sha512>`__)                            |
++ RPM (CentOS)  + 6 or greater +--------------+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
RPM is not only CentOS: other distros like Fedora, openSUSE, AL, OL use it as package manager.
IMHO RPM column should said "RPM" and "Version" should contain OSes to use that package